articles/web-dev's-guide-SEO/index.md

```diff
@@ -0,0 +1,85 @@
+   <meta name="keywords" content="SEO, Web dev, pagerank, node.js "/>
+    ```
+    The description and keywords are two such options that crawlers look for. Keywords are not used much today, but it's a good practice to include them. 
+
+2. **Adding title tag**: The title tag displays the current title of the webpage. A common mistake that developers make is having the same title tag for most of the pages on a website. The ranking algorithms penalize such behavior of websites for lack of information. The title tag must be modified according to the content at hand. For example, look at the current tab. Is the title " Developer's Guide to Search Engine Optimization| Section"? Such optimization is required for getting better results at search engines.   
+
+3. **Adding favicons:** Favicons are the tiny icons that appear on the tabs. They don't directly affect SEO. They play a vital role in optimizing the user experience. The favicons represent the brand and therefore have huge importance. Looking at the f icon on top of the Facebook page helps us easily identify the tab amidst other tabs. The favicon also helps to identify the webpage in bookmark lists as well as search history. Better and easier identification of websites enables larger time spent on the site thereby optimizing it.
+
+### Social Metadata
+As discussed earlier, controlling the experience of the user upon the clicking of an URL is very important. Various technologies and protocols are built to enhance the same. 
+
+#### Facebook Open Group Protocol
+
+Facebook's [Open Graph Protocol](https://developers.facebook.com/docs/sharing/opengraph/) decides how URLs are shown in Facebook posts. There are various meta tags that one can add. It is the same as before, except a property tag is added.
+
+```html
+<meta property="og:proprty_name" content="canonicalURL">
+```
+og stands for Open Graph. The ```property_name``` mentioned in meta tag after og takes several values. They are as follows:
+1. title
+2. type
+3. URL
+4. image
+5. admins
+6. site_name
+7. description 
+
+Let us look at how these property names help enhance the user experience in a Facebook post.
+![open graph protocol](opengraph.jpg)
+
+*Source: https://2.bp.blogspot.com/-kzdcNpZkkK0/VBHOHfDjlkI/AAAAAAAAALs/Ud6T0JfaB4A/s1600/SharedLink.png*
+
+The ```canonicalURL``` mentioned in the meta tag is an alias for human-readable URLs. Let's understand it in detail. When we go to a link on amazon's website, we find that the page URL has the book's name in it and some other details as well. On the other hand, links that are shared are usually numeric. The numeric data is the non-changeable URL, which is called the canonical URL. The URL with descriptive words in it is called the Fetched URL. The name of the book may change which may lead to a change in the fetched URL. The canonical URL remains the same for a given product. It is a unique value derived using the primary key from the database. 
```

Comment:
You only need one backtick for inline code formatting. Three is for code blocks.